rfc/training-content-delivery.md

```diff
@@ -0,0 +1,75 @@
+
+This approach leverages our existing toolchains (so it's familiar to create and iterate on content), as well as ensures that the content leverages our current branding, so it looks professional and cohesive. A secondary benefit is that it familiarizes users with Cockroach Labs' design language, making it easier to navigate through our other properties, as well.
+
+Because of this approach's integration with our existing workflow, we can easily achieve it by placing this content in a subdirectory off of `cockroachlabs.com`, using a similar approach to both `/docs` and `/blog`.
+
+# Motivation
+
+We've begun developing training curriculum but have not yet determined where it lives, or how to best deliver it to attendees. Because this approach could drastically alter the authoring process, it's best to determine how this is done before we make substantial headway into producing these deliverables.
+
+By the end of this RFC, there will be a plan for the method we plan to use for delivering training content.
+
+# Guide-level explanation
+
+### Elements
+
+CockroachDB training is comprised of 3 elements:
+
+- **Presentations** give the instructor a tool to make sure the class follows the outlined curriculum, as well as reinforce topics discussed for students with a quick summary to help them mentally model the content, as well as a canvas for visual aids.
+- **Labs** provide attendees a chance to work with CockroachDB, using the information they just learned in the presentation. This should include a mixture of reinforcement of content form the presentation and step-by-step instructions to accomplish a task directly related to the presentation.
+- **Demos** give the instructor the option to demonstrate something that might be difficult for attendees to quickly reproduce themselves, e.g. anomalous reads from something like Jepsen. These can be "staged" labs to help students understand that something is true or real without having to relate the conditions for it.
+
+### Delivery
+
+To deliver the material, each element has its own needs:
+
+- Presentations require there being only a single bit of text presented at a time. To achieve this, I recommend using Google Slides.
+- Labs require much more text, as well as other hypertext tools that we've already developed on `/docs`. To achieve this, I recommend using Markdown rendered into HTML through Jekyll.
+- Demos each have different needs, but are often broadcast in the classroom from the instructor's computer. I recommend either recording demos that can be shown on YouTube, or solving demos on a case-by-case basis.
+
+### Structure
+
+As a wrapper around all of this content, the training material is hosted on `cockroachlabs.com/training`. We currently do not have password-restricted access on this content, but may in the future.
```

Comment:
As discussed offline, it's fine for this content to be public; when used in classrooms, the students get added value from guided exercises and the instructors' expertise.